src/field_theory/intermediate_field.lean

```diff
@@ -377,43 +376,45 @@ section tower
 
 /-- Lift an intermediate_field of an intermediate_field -/
 def lift1 {F : intermediate_field K L} (E : intermediate_field K F) : intermediate_field K L :=
```

Comment:
Let's not reproduce the `vector3` naming weirdness!
```suggestion
def lift {F : intermediate_field K L} (E : intermediate_field K F) : intermediate_field K L :=
```

Reply:
Nothing even uses this; shall I just remove it?

Reply:
I agree that we probably should get rid of it, but @kmill just introduced the `subgraph` analogues of it.

Reply:
I'm not sure the subgraph-of-subgraph to subgraph map is a perfect analogue (I feel that combinatorial objects have somewhat different considerations from algebraic structures), but for what it's worth I've been making these sorts of shortcuts be `@[reducible]`.

I'd vote don't remove it, and perhaps write add to the docstring to consider writing `S.map val` directly? That way someone trying to figure out how to do this lift can learn how to do it more easily.

Reply:
Sounds like enough of a resistance that removing it is out of scope of this PR.

I'll remove the 1 suffix for now, and propose a full removal in another PR.